src/catalogue/items/catalogueItemsDialog.component.tsx

```diff
@@ -850,11 +851,19 @@ function CatalogueItemsDialog(props: CatalogueItemsDialogProps) {
                           {property.type === 'boolean' ? (
                             <FormControl fullWidth>
                               <Autocomplete
+                                disableClearable={property.mandatory ?? false}
                                 id={`catalogue-item-property-${property.name.replace(
                                   /\s+/g,
                                   '-'
                                 )}`}
-                                value={(propertyValues[index] as string) ?? ''}
+                                value={
+                                  propertyValues[index]
+                                    ? (propertyValues[index] as string)
+                                        .charAt(0)
+                                        .toUpperCase() +
+                                      (propertyValues[index] as string).slice(1)
+                                    : ''
+                                }
```

Comment:
![image](https://github.com/ral-facilities/inventory-management-system/assets/144466888/78ce8a24-cda3-4a54-9720-1c6d63a084ca)

This causes the value in the field to be different from the label, I can make the labels lowercase so that they match or I can leave as is